### ISSUE TITLE
App crash when Cancel button tapped on CVC popup

### ISSUE THREAD

**Issue by BartlomiejSopalaKarhoo** [Adyen/adyen-ios#869]
Hi.
We have a problem, our app crashes when we click Cancel on Adyen popup with CVC code and stack trace shows that it is happening inside Adyen library. The same happens when we try to add naw card.

**To Reproduce**
1. Open CVC code popup
2. Click on 'Cancel'
3. See error

**Screenshots**
<img width="755" alt="Screenshot 2022-04-27 at 13 54 35" src="https://user-images.githubusercontent.com/97114085/165513636-b3529f88-4e67-4ddf-9126-9769dee0c9c5.png">
<img width="910" alt="Screenshot 2022-04-27 at 13 53 48" src="https://user-images.githubusercontent.com/97114085/165513656-cc0035ea-749b-47aa-bd23-9a1d174ebfb9.png">

**Smartphone (please complete the following information):**
 - Device: [1Phone 13 Pro Max  - simulator]
 - OS: [iOS 15.4]
 - SDK Version [4.7.1 integrated via Swift Package Manager]

**Comment by descorp** [comment]
Hey @BartlomiejSopalaKarhoo 

Thanks for reaching out!
Are you using storyboards / Interface builder?

We do not use .xib, and we do not have "CardView" class in our SDK. 🤔

**Comment by tarikalaouimhamdi** [comment]
Hello,
we have a start on this problem.
Indeed, the CardViewController inherits from the FormViewController, which at its creation, instantiates super.init(nibName: nil, bundle: nil) 
Except that according to Apple's documentation, if you don't specify an Xib at initialization, it will automatically look for an xib with the class name if it finds it, and this is the case in our projet.

please see : https://developer.apple.com/documentation/uikit/uiviewcontroller/1621487-nibname

```
If the view controller class name ends with the word ‘Controller’, as in MyViewController, it looks for a nib file whose name matches the class name without the word ‘Controller’, as in MyView.nib.

It looks for a nib file whose name matches the name of the view controller class. For example, if the class name is MyViewController, it looks for a MyViewController.nib file.
```

**Comment by tarikalaouimhamdi** [comment]
To fix this, you have to specify the bundle AdyenScope, for example : 

`super.init(nibName: nil, bundle: Bundle.coreInternalResources)`

**Comment by descorp** [comment]
Thanks @talaoui-accor !

This is great insight!

**Comment by neelSharma12** [comment]
PR with a fix has been merged and released in version 4.7.3 for this hence,I am closing this ticket.
